test/e2e/service.go

```diff
@@ -1117,25 +1117,25 @@ var _ = framework.KubeDescribe("Services", func() {
 		By("reading clientIP using the TCP service's NodePort")
 		content := jig.GetHTTPContent(nodeIP, tcpNodePort, kubeProxyLagTimeout, "/clientip")
 		clientIP := content.String()
-		framework.Logf("ClientIP detected by target pod using NodePort is %s", clientIP)
+		jig.Logf("ClientIP detected by target pod using NodePort is %s", clientIP)
 		By("reading clientIP using the TCP service's service port via its external VIP")
 		content = jig.GetHTTPContent(ingressIP, svcTcpPort, kubeProxyLagTimeout, "/clientip")
 		clientIP = content.String()
-		framework.Logf("ClientIP detected by target pod using VIP:SvcPort is %s", clientIP)
+		jig.Logf("ClientIP detected by target pod using VIP:SvcPort is %s", clientIP)
 		By("checking if Source IP is preserved")
 		if strings.HasPrefix(clientIP, "10.") {
-			framework.Failf("Source IP was NOT preserved")
+			jig.Failf("Source IP was NOT preserved")
 		}
 		By("finding nodes for all service endpoints")
 		endpoints, err := c.Endpoints(namespace).Get(serviceName)
 		if err != nil {
-			framework.Failf("Get endpoints for service %s/%s failed (%s)", namespace, serviceName, err)
+			jig.Failf("Get endpoints for service %s/%s failed (%s)", namespace, serviceName, err)
 		}
 		if len(endpoints.Subsets[0].Addresses) == 0 {
-			framework.Failf("Expected Ready endpoints - found none")
+			jig.Failf("Expected Ready endpoints - found none")
 		}
 		readyHostName := *endpoints.Subsets[0].Addresses[0].NodeName
```

Comment:
readyHostName is confusing, makes me think it's the name of a ready node but it's really the name of a node a ready endpoint is on. Suggest endpointNodeName.